Formula/c/cppinsights.rb

```diff
@@ -24,7 +24,31 @@ class Cppinsights < Formula
   end
 
   def install
-    ENV.llvm_clang if OS.mac? && DevelopmentTools.clang_build_version <= 1500
+    if OS.mac? && DevelopmentTools.clang_build_version <= 1500
+      ENV.llvm_clang
+      ENV["CXXFLAGS"] = "--warning-suppression-mappings=#{buildpath}/suppression_mappings.txt"
+      (buildpath/"suppression_mappings.txt").write <<~EOF
+        [old-style-cast]
+        src:#{Formula["llvm"].opt_include.realpath}/*
+        [unused-parameter]
+        src:#{Formula["llvm"].opt_include.realpath}/*
+      EOF
```

Comment:
Do we really need to suppress warnings? Is that huge?

Reply:
@daeho-ro The warnings cause a build failure. I think the alternative approaches are:

- prevent these warnings from being treated as errors (and causing the build to fail)
- resolve warnings
- suppress warnings

Reply:
I don't think resolving the warnings is feasible, since the issue is within the LLVM headers. And disabling "warnings as errors" more generally in this build might be undesirable (as I suspect the project maintainers intended this behavior). I figured suppressing only these warnings only for the LLVM headers was the least invasive approach.